client/index.jsx

```diff
@@ -1,20 +1,40 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import $ from 'jquery';
```

Comment:
You might consider refactoring to a different method than you have before, just for the experience. If you haven't used Axios or the built-in Fetch API, this is a low-pressure time to try them out.

There's nothing wrong with jQuery being used here, to be clear. This is a process suggestion, not something requiring changes.

Reply:
Thanks! I considered trying axios and will most likely convert for the experience after completing deliverables.